### DISCUSSION TITLE
Why is the PREFIX variable pointing to "/data/data/com.termux/files/usr" instead of "/data/data/com.termux/files" ?

### DISCUSSION THREAD

**Discussion by sebma** [termux/termux-app#2461] [General]
Hi,
I don't really understand why is the PREFIX variable pointing to `/data/data/com.termux/files/usr` instead of `/data/data/com.termux/files` : 
```bash
$ echo $PREFIX
/data/data/com.termux/files/usr
$ ls -F /data/data/com.termux/files
bin/  data/  dev/  etc/  home/  lib/  proc/  root/  sbin/  share/  storage/  system/  tmp/  usr/  var/  vendor/
```

As the `dev/`, `home/`, `proc/`, `root/`, `var/` directories are located in `/data/data/com.termux/files` instead of `/data/data/com.termux/files/usr`, should not the PREFIX variable be pointing to `/data/data/com.termux/files instead` ?

**Comment by thunder-coding** [comment]
I don't know what level of mess you have done to your Termux environment, but I want to tell you that `/data/data/com.termux/files` only contains teo directories namely `usr/`(equivalent to /usr) and `home/`(equivalent to /home).

The `PREFIX` environment variable points to `/data/data/com.termux/files/usr`

Probably you need a fresh reinstallation of Termux

**Comment by sebma** [comment]
@thunder-coding I think the `termux-chroot` tool created these directories via the `proot` command : 
```bash
$ man proot | col -b | sed -n "/-r path/,/recommended/p"
       -r path, --rootfs=path
	      Use path as the new guest root file-system, default is /.

	      The specified path typically contains a Linux distribution where
	      all new programs will be confined.  The default rootfs is / when
	      none is specified, this makes sense when the bind mechanism is
	      used to relocate host files and directories, see the -b option
	      and the Examples section for details.

	      It is recommended to use the -R or -S options instead.
$ bash -x termux-chroot "echo WHATEVER" 2>&1 | grep -w exec.*proot | grep -o -- "-r [^ ]*"
-r /data/data/com.termux/files/usr/..
$
```
As you can see this `proot` call has `-r path, --rootfs` point to `/data/data/com.termux/files/usr/..` that is to say `/data/data/com.termux/files/`.
